README.md

```diff
@@ -4,6 +4,30 @@
 
 Containerized [**Tizonia**](http://www.tizonia.org/) cloud music player that uses the host's sound system.
 
+## Mac Support
```

Comment:
Howdy
thanks for trying this out on mac; i can confirm that it works
i assembled the following script so we would automate these steps; hope it eases someone else's setup

```sh
#get pulse audio
brew install pulseaudio

#uncomment load-module module-esound-protocol-tcp and
sed -i.bak -e 's/^#load-module module-esound-protocol-tcp/load-module module-esound-protocol-tcp/g' -e 's/^#load-module module-native-protocol-tcp/load-module module-native-protocol-tcp/g'  '/usr/local/Cellar/pulseaudio/13.0/etc/pulse/default.pa'

#list sinks with description
pactl list sinks | egrep '(^\s+(Name|Description).*$)|^Sink.*$'

echo 'Pick an output device enter the numer ( enter a number between 1 and' $(pactl list sinks short | wc -l) ')'
read outputChannel

pacmd set-default-sink $outputChannel
brew services start pulseaudio
cp ./docker-tizonia /usr/local/bin  

./docker-tizonia

chmod a+wrx $HOME/.config/tizonia
```